include/dsAudio.h

```diff
@@ -211,52 +211,6 @@ dsError_t  dsAudioPortTerm();
  */
 dsError_t  dsGetAudioPort(dsAudioPortType_t type, int index, intptr_t *handle);
 
```

Comment:
dsSetStereoMode - function used to set the output port format, update the function description

Reply:
Its the Output Mode - PCM, Passthrough , etc. Needs to be discussed with Module owners